metricbeat/module/kafka/partition/_meta/fields.yml

```diff
@@ -16,14 +16,53 @@
           type: long
           description: >
             Oldest offset of the partition.
+        - name: error
+          type: long
+          description: >
+            Error code from fetching offset.
+
     - name: partition
+      type: group
+      description: >
+        Partition data.
+      fields:
+        - name: id
+          type: long
+          description: >
+            Partition id.
+
+        - name: leader
+          type: long
+          description: >
+            Leader id (broker).
+        - name: isr
+          type: list
+          description: >
+            List of isr ids.
+        - name: replica
+          type: long
+          description: >
+            Replica id (broker).
+
+        - name: insync_replica
+          type: boolean
+          description: >
+            Indicates if replica is included in the in-sync replicate set (ISR).
+
+        - name: error
```

Comment:
Hm, if we switch this one to error_code, we would have to change the other 2 errors too. WDYT?